### DISCUSSION TITLE
osu!catch editor needs snapping lines on the playfield

### DISCUSSION THREAD

**Discussion by Xinnoh** [ppy/osu#23462] [Ideas] [answered]
Tried the catch editor, couldn't use it. When placing objects it's unreasonably hard to know where in the timeline you're placing stuff.

Malody editor looks like this which I was fine with using before.
![Screenshot 2023-05-09 033236](https://github.com/ppy/osu/assets/30382015/ac26cf57-3cf9-4b43-a72e-7be1f51cee5e)

![Screenshot 2023-05-11 013756](https://github.com/ppy/osu/assets/30382015/e72515b7-96c9-42db-bfde-a8658d825caf)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23770
